CompletedProcess(args='slither  ./smartbugs-curated/.~lock._versao.xlsx# --json ./slither/smartbugs-curated/json/.~lock._versao.xlsx#.json', returncode=1, stdout='', stderr='Traceback (most recent call last):\n  File "/home/felipe/.local/bin/slither", line 8, in <module>\n    sys.exit(main())\n  File "/home/felipe/.local/lib/python3.10/site-packages/slither/__main__.py", line 727, in main\n    main_impl(all_detector_classes=detectors, all_printer_classes=printers)\n  File "/home/felipe/.local/lib/python3.10/site-packages/slither/__main__.py", line 833, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File "/home/felipe/.local/lib/python3.10/site-packages/slither/__main__.py", line 96, in process_all\n    compilations = compile_all(target, **vars(args))\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/crytic_compile.py", line 718, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/crytic_compile.py", line 207, in __init__\n    self._compile(**kwargs)\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/crytic_compile.py", line 629, in _compile\n    self._platform.compile(self, **kwargs)\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/platform/solc.py", line 151, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/platform/solc.py", line 280, in _get_targets_json\n    return _run_solc(\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/platform/solc.py", line 494, in _run_solc\n    raise InvalidCompilation(f"{filename} is not the expected format \'.sol\'")\ncrytic_compile.platform.exceptions.InvalidCompilation: ./smartbugs-curated/.~lock._versao.xlsx# is not the expected format \'.sol\'\n')
CompletedProcess(args='slither  ./smartbugs-curated/0x01f8c4e3fa3edeb29e514cba738d87ce8c091d3f.sol --json ./slither/smartbugs-curated/json/0x01f8c4e3fa3edeb29e514cba738d87ce8c091d3f.sol.json', returncode=1, stdout='', stderr='\'solc --version\' running\nTraceback (most recent call last):\n  File "/home/felipe/.local/bin/slither", line 8, in <module>\n    sys.exit(main())\n  File "/home/felipe/.local/lib/python3.10/site-packages/slither/__main__.py", line 727, in main\n    main_impl(all_detector_classes=detectors, all_printer_classes=printers)\n  File "/home/felipe/.local/lib/python3.10/site-packages/slither/__main__.py", line 833, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File "/home/felipe/.local/lib/python3.10/site-packages/slither/__main__.py", line 96, in process_all\n    compilations = compile_all(target, **vars(args))\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/crytic_compile.py", line 718, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/crytic_compile.py", line 207, in __init__\n    self._compile(**kwargs)\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/crytic_compile.py", line 629, in _compile\n    self._platform.compile(self, **kwargs)\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/platform/solc.py", line 151, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/platform/solc.py", line 280, in _get_targets_json\n    return _run_solc(\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/platform/solc.py", line 497, in _run_solc\n    compiler="solc", version=get_version(solc, env), optimized=is_optimized(solc_arguments)\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/platform/solc.py", line 395, in get_version\n    raise InvalidCompilation(\ncrytic_compile.platform.exceptions.InvalidCompilation: \nSolidity version not found:\nSTDOUT:\n\nSTDERR:\nTraceback (most recent call last):\n  File "/home/felipe/.local/bin/solc", line 8, in <module>\n    sys.exit(solc())\n  File "/home/felipe/.local/lib/python3.10/site-packages/solc_select/__main__.py", line 87, in solc\n    res = current_version()\n  File "/home/felipe/.local/lib/python3.10/site-packages/solc_select/solc_select.py", line 67, in current_version\n    raise argparse.ArgumentTypeError(\nargparse.ArgumentTypeError: No solc version set. Run `solc-select use VERSION` or set SOLC_VERSION environment variable.\n\n')
CompletedProcess(args='slither  ./smartbugs-curated/0x07f7ecb66d788ab01dc93b9b71a88401de7d0f2e.sol --json ./slither/smartbugs-curated/json/0x07f7ecb66d788ab01dc93b9b71a88401de7d0f2e.sol.json', returncode=1, stdout='', stderr='\'solc --version\' running\nTraceback (most recent call last):\n  File "/home/felipe/.local/bin/slither", line 8, in <module>\n    sys.exit(main())\n  File "/home/felipe/.local/lib/python3.10/site-packages/slither/__main__.py", line 727, in main\n    main_impl(all_detector_classes=detectors, all_printer_classes=printers)\n  File "/home/felipe/.local/lib/python3.10/site-packages/slither/__main__.py", line 833, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File "/home/felipe/.local/lib/python3.10/site-packages/slither/__main__.py", line 96, in process_all\n    compilations = compile_all(target, **vars(args))\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/crytic_compile.py", line 718, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/crytic_compile.py", line 207, in __init__\n    self._compile(**kwargs)\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/crytic_compile.py", line 629, in _compile\n    self._platform.compile(self, **kwargs)\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/platform/solc.py", line 151, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/platform/solc.py", line 280, in _get_targets_json\n    return _run_solc(\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/platform/solc.py", line 497, in _run_solc\n    compiler="solc", version=get_version(solc, env), optimized=is_optimized(solc_arguments)\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/platform/solc.py", line 395, in get_version\n    raise InvalidCompilation(\ncrytic_compile.platform.exceptions.InvalidCompilation: \nSolidity version not found:\nSTDOUT:\n\nSTDERR:\nTraceback (most recent call last):\n  File "/home/felipe/.local/bin/solc", line 8, in <module>\n    sys.exit(solc())\n  File "/home/felipe/.local/lib/python3.10/site-packages/solc_select/__main__.py", line 87, in solc\n    res = current_version()\n  File "/home/felipe/.local/lib/python3.10/site-packages/solc_select/solc_select.py", line 67, in current_version\n    raise argparse.ArgumentTypeError(\nargparse.ArgumentTypeError: No solc version set. Run `solc-select use VERSION` or set SOLC_VERSION environment variable.\n\n')
CompletedProcess(args='slither  ./smartbugs-curated/0x0cbe050f75bc8f8c2d6c0d249fea125fd6e1acc9.sol --json ./slither/smartbugs-curated/json/0x0cbe050f75bc8f8c2d6c0d249fea125fd6e1acc9.sol.json', returncode=1, stdout='', stderr='\'solc --version\' running\nTraceback (most recent call last):\n  File "/home/felipe/.local/bin/slither", line 8, in <module>\n    sys.exit(main())\n  File "/home/felipe/.local/lib/python3.10/site-packages/slither/__main__.py", line 727, in main\n    main_impl(all_detector_classes=detectors, all_printer_classes=printers)\n  File "/home/felipe/.local/lib/python3.10/site-packages/slither/__main__.py", line 833, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File "/home/felipe/.local/lib/python3.10/site-packages/slither/__main__.py", line 96, in process_all\n    compilations = compile_all(target, **vars(args))\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/crytic_compile.py", line 718, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/crytic_compile.py", line 207, in __init__\n    self._compile(**kwargs)\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/crytic_compile.py", line 629, in _compile\n    self._platform.compile(self, **kwargs)\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/platform/solc.py", line 151, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/platform/solc.py", line 280, in _get_targets_json\n    return _run_solc(\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/platform/solc.py", line 497, in _run_solc\n    compiler="solc", version=get_version(solc, env), optimized=is_optimized(solc_arguments)\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/platform/solc.py", line 395, in get_version\n    raise InvalidCompilation(\ncrytic_compile.platform.exceptions.InvalidCompilation: \nSolidity version not found:\nSTDOUT:\n\nSTDERR:\nTraceback (most recent call last):\n  File "/home/felipe/.local/bin/solc", line 8, in <module>\n    sys.exit(solc())\n  File "/home/felipe/.local/lib/python3.10/site-packages/solc_select/__main__.py", line 87, in solc\n    res = current_version()\n  File "/home/felipe/.local/lib/python3.10/site-packages/solc_select/solc_select.py", line 67, in current_version\n    raise argparse.ArgumentTypeError(\nargparse.ArgumentTypeError: No solc version set. Run `solc-select use VERSION` or set SOLC_VERSION environment variable.\n\n')
CompletedProcess(args='slither  ./smartbugs-curated/0x19cf8481ea15427a98ba3cdd6d9e14690011ab10.sol --json ./slither/smartbugs-curated/json/0x19cf8481ea15427a98ba3cdd6d9e14690011ab10.sol.json', returncode=1, stdout='', stderr='\'solc --version\' running\nTraceback (most recent call last):\n  File "/home/felipe/.local/bin/slither", line 8, in <module>\n    sys.exit(main())\n  File "/home/felipe/.local/lib/python3.10/site-packages/slither/__main__.py", line 727, in main\n    main_impl(all_detector_classes=detectors, all_printer_classes=printers)\n  File "/home/felipe/.local/lib/python3.10/site-packages/slither/__main__.py", line 833, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File "/home/felipe/.local/lib/python3.10/site-packages/slither/__main__.py", line 96, in process_all\n    compilations = compile_all(target, **vars(args))\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/crytic_compile.py", line 718, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/crytic_compile.py", line 207, in __init__\n    self._compile(**kwargs)\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/crytic_compile.py", line 629, in _compile\n    self._platform.compile(self, **kwargs)\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/platform/solc.py", line 151, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/platform/solc.py", line 280, in _get_targets_json\n    return _run_solc(\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/platform/solc.py", line 497, in _run_solc\n    compiler="solc", version=get_version(solc, env), optimized=is_optimized(solc_arguments)\n  File "/home/felipe/.local/lib/python3.10/site-packages/crytic_compile/platform/solc.py", line 395, in get_version\n    raise InvalidCompilation(\ncrytic_compile.platform.exceptions.InvalidCompilation: \nSolidity version not found:\nSTDOUT:\n\nSTDERR:\nTraceback (most recent call last):\n  File "/home/felipe/.local/bin/solc", line 8, in <module>\n    sys.exit(solc())\n  File "/home/felipe/.local/lib/python3.10/site-packages/solc_select/__main__.py", line 87, in solc\n    res = current_version()\n  File "/home/felipe/.local/lib/python3.10/site-packages/solc_select/solc_select.py", line 67, in current_version\n    raise argparse.ArgumentTypeError(\nargparse.ArgumentTypeError: No solc version set. Run `solc-select use VERSION` or set SOLC_VERSION environment variable.\n\n')
